clients/discovery_test.go

```diff
@@ -1,13 +1,22 @@
 package clients
 
 import (
-	"io/ioutil"
+	_ "embed"
 	"terraform-provider-ome/models"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	//go:embed json_data/payloadCreateDiscovery.json
+	jsonData1 []byte
```

Comment:
Please give these variables proper names (the name of the file being embedded is a good starting point). All else is good.

Reply:
"payloadCreateDiscovery []byte"
does this look good?

Reply:
sure sure